bpf/lib/egress_policies.h

```diff
@@ -34,25 +44,16 @@ is_cluster_destination(struct iphdr *ip4, __u32 dst_id, __u32 tunnel_endpoint)
 	return false;
 }
 
-/* EGRESS_STATIC_PREFIX gets sizeof non-IP, non-prefix part of egress_key */
-# define EGRESS_STATIC_PREFIX							\
-	(8 * (sizeof(struct egress_key) - sizeof(struct bpf_lpm_trie_key)	\
-	      - 4))
-# define EGRESS_PREFIX_LEN(PREFIX) (EGRESS_STATIC_PREFIX + (PREFIX))
-# define EGRESS_IPV4_PREFIX EGRESS_PREFIX_LEN(32)
-
-static __always_inline __maybe_unused struct egress_info *
-egress_lookup4(const void *map, __be32 sip, __be32 dip)
+static __always_inline
+struct egress_gw_policy_entry *lookup_ip4_egress_gw_policy(__be32 saddr, __be32 daddr)
 {
-	struct egress_key key = {
+	struct egress_gw_policy_key key = {
 		.lpm_key = { EGRESS_IPV4_PREFIX, {} },
-		.sip = sip,
-		.dip = dip,
+		.saddr = saddr,
+		.daddr = daddr,
 	};
-	return map_lookup_elem(map, &key);
+	return map_lookup_elem(&EGRESS_POLICY_MAP, &key);
 }
 
-# define lookup_ip4_egress_endpoint(sip, dip) \
-	egress_lookup4(&EGRESS_MAP, sip, dip)
```

Comment:
FYI, I'll have to revert all changes to this file in the SRv6 PR, except for the map/key name changes.